testgrid/config.yaml

```diff
@@ -7145,6 +7140,11 @@ dashboards:
   - name: pull-kubernetes-node-e2e-alpha
     test_group_name: pull-kubernetes-node-e2e-alpha
     base_options: width=10
+  - name: pull-kubernetes-e2e-kops-aws
+    test_group_name: pull-kubernetes-e2e-kops-aws
+    base_options: width=10
+    alert_options:
+      alert_mail_to_addresses: michelle.192837+alerts@gmail.com, lusensjtu+alerts@gmail.com
```

Comment:
an aside: this is still @michelle192837 & @krzyzacy's personal email addresses from testing the alerting functionality long ago, we should do a cleanup pass to remove these later.

Reply:
wait so we have @krzyzacy's personal email now?

Reply:
:smirk: